<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QMainWindow" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>address.svg</normaloff>address.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,13">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../search.svg</normaloff>../search.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../close.svg</normaloff>../close.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>file</string>
    </property>
    <widget class="QMenu" name="menuAdd_data">
     <property name="title">
      <string>add data</string>
     </property>
     <addaction name="actionGeojson"/>
     <addaction name="actionShp"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuAdd_data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout_us"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>frame</string>
    </property>
    <addaction name="actionContent"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>database</string>
    </property>
    <addaction name="actionSave_shapefile"/>
    <addaction name="actionGet_databaseData"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuDatabase"/>
   <addaction name="menu_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGeojson">
   <property name="text">
    <string>geojson</string>
   </property>
  </action>
  <action name="actionShp">
   <property name="text">
    <string>shp</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>about us</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="text">
    <string>content</string>
   </property>
  </action>
  <action name="actionSave_shapefile">
   <property name="text">
    <string>save shapefile</string>
   </property>
  </action>
  <action name="actionGet_databaseData">
   <property name="text">
    <string>get databaseData</string>
   </property>
  </action>
  <action name="actionIndex_grid">
   <property name="text">
    <string>showIndexGrid</string>
   </property>
  </action>
  <action name="actionHideIndexGrid">
   <property name="text">
    <string>hideIndexGrid</string>
   </property>
  </action>
  <action name="actionKDE">
   <property name="text">
    <string>KDE</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MyClassClass</receiver>
   <slot>search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
  <slot>restore2()</slot>
 </slots>
</ui>
